release-notes/v1_6.md

```diff
@@ -7,70 +7,88 @@ MetaDescription: See what is new in the Visual Studio Code September 2016 Releas
+### Detecting a corrupt installation
 
-![Icons](images/1_6/electron.png)
+We have found ourselves spending many hours investigating VS Code issues that were ultimately caused by corrupt VS Code installations. Some extensions directly modify (patch) the VS Code product in such a way that is semi-permanent (until the next update) and this can cause hard to reproduce issues.
 
-### Release notes inside VS Code
+We have therefore added a background check where we try to detect if the VS Code bits have been changed on disk. We are not trying to block VS Code patching, but we want to raise awareness that patching VS Code means you are running an unsupported version.
 
-The release notes can now be opened right within VS Code. Use the global **Show Release Notes** command or the **Help** > **Release Notes** menu to access them.
+![corrupt install](images/1_6/corrupt-install.png)
 
-### Quickly switch between running VS Code instances
 
-A new command **Switch Window** was added that opens a picker showing all the opened VS Code windows (instances) and provides a fast way to switch to a specific window right from VS Code.
+### Issue reporting improvement
+
+Related to this, we often investigate issues that are caused by an installed extension. To help us with these investigations, we have enriched the **Help** > **Report Issues** action to seed the issue description with the list of installed extensions.
 
-### Image dimensions in Status Bar
+Please use this action when reporting issues so we have more information to help track down the problem and ultimately resolve the issue for you quicker.
+
+
+### Image dimensions & Binary file size
 
 When you open a binary file in VS Code, extra meta information is now displayed in the Status Bar. For all binary files, we show the size of the file and for images, we also show the image dimensions.
 
 ![Status](images/1_6/status.png)
 
-VS Code also now detects image file changes on disk so you no longer see stale old versions of an image.
+We also made a few tweeks so that VS Code also now detects image file changes on disk so you no longer see stale old versions of an image.
+
+
 
 ### New location for dirty indicator when Tabs are disabled
 
 We decided to move the dirty (unsaved file) indicator over to the **Close** button when Tabs (tabbed headers) are disabled. This makes the experience consistent with other places where we replace the Close button with the dirty indicator (for example, inside **OPEN EDITORS** or when Tabs are enabled).
 
 ![Status](images/1_6/dirty.png)
 
-### Detecting a corrupt installation
 
-We have found ourselves spending many hours investigating VS Code issues that were ultimately caused by corrupt VS Code installations. Some extensions directly modify (patch) the VS Code product in such a way that is semi-permanent (until the next update) and this can cause hard to reproduce issues.
+### Improving command discoverability
```

Comment:
Part of me felt we could skip this section but I think it's OK - it's just that for some people they won't see it.  I therefore attempted to refactor to explain why were were doing it - our goals